src/ui/components/NetworkMonitor/useNetworkContextMenu.tsx

```diff
@@ -46,30 +37,7 @@ export default function useNetworkContextMenu({
 
   const addComment = () => {
     startTransition(async () => {
-      if (showCommentsPanel !== null) {
-        showCommentsPanel();
-      }
-
-      const typeData = await createTypeDataForNetworkRequestComment(
-        requestSummary.id,
-        requestSummary.method,
-        requestSummary.name,
-        requestSummary.status ?? null,
-        beginTime,
-        requestSummary.type
-      );
-
-      await addCommentGraphQL(graphQLClient, accessToken!, recordingId, {
-        content: "",
-        hasFrames: true,
-        isPublished: false,
-        point: requestSummary.triggerPoint?.point,
-        time: requestSummary.triggerPoint?.time,
-        type: COMMENT_TYPE_NETWORK_REQUEST,
-        typeData,
-      });
-
-      invalidateCache();
+      dispatch(createNetworkRequestComment(requestSummary, recordingId));
```

Comment:
Technically this is a bugfix that I noticed along the way while fixing an e2e test.